features/attached_commands.feature

```diff
@@ -213,7 +214,7 @@ Feature: Command behaviour when attached to an UA subscription
             """
             This command must be run as root \(try using sudo\).
             """
-        When I run `ua auto-attach` with sudo
+        Then I verify that running `ua auto-attach` `with sudo` exits `2`
```

Comment:
Why did we change the `When` clause for the `Then` clause here ?

Reply:
:man_facepalming: